pkg/grafana/sso_settings.go

```diff
@@ -14,28 +14,27 @@ const (
 	grafanaAdminRole  = "Admin"
 	grafanaEditorRole = "Editor"
 	grafanaViewerRole = "Viewer"
+
+	ssoProvider = "generic_oauth"
 )
 
 func (s *Service) ConfigureSSOSettings(ctx context.Context, organizations []Organization) error {
 	logger := log.FromContext(ctx)
 
-	provider := "generic_oauth"
-	resp, err := s.grafanaAPI.SsoSettings.GetProviderSettings(provider, nil)
+	resp, err := s.grafanaAPI.SsoSettings.GetProviderSettings(ssoProvider, nil)
 	if err != nil {
 		logger.Error(err, "failed to get sso provider settings.")
 		return errors.WithStack(err)
 	}
 
 	orgsMapping := generateGrafanaOrgsMapping(organizations)
-	settings := resp.Payload.Settings.(map[string]interface{})
-	settings["role_attribute_path"] = "to_string('Viewer')"
```

Comment:
Falls back to the value defined in config because it is then hard to know where to change what